src/Kros.AspNetCore/Environments/EnvironmentsExt.cs

```diff
@@ -1,28 +1,28 @@
-﻿namespace Microsoft.AspNetCore.Hosting
+﻿namespace Microsoft.Extensions.Hosting
 {
     /// <summary>
     /// Valid environments.
     /// </summary>
-    public static class Environments
```

Comment:
Class name the same as file name. Rename it to `EnvironmentsEx` (_Ex_ is kind of a standard in Microsoft sources).